app/models/concerns/normalize_city_name.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module NormalizeCityName
```

Comment:
I had to rename this to be less generic `NormalizeColumn` => `NormalizeCityName` because on line 29 below, we use `cities_mapping.yml` to fetch incorrect to correct city names mapping.